src/components/WeekTabs/WeekTabs.jsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { useEffect, useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+
+import { TabsBackground, TabsList, TabsListItem, DayTitle, WeekData } from "./WeekTabsStyled";
+import { getDayTitle } from "../../utilities/MainUtils";
+
+const WEEK_DAYS = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"];
+const WEEKDAY_VALUES = new Map();
+
+export const WeekTabs = ({weekDataString, selectedDay, setSelectedDay, today, setSelectedDate}) => {
+
+    const [isFullDayNames, setIsFullDayNames] = useState(true);
+    const navigate = useNavigate();
+
+    const setDayNamesLength = () => {
+        if(Math.abs(document.querySelector("body").getBoundingClientRect().width > 768)){
+            setIsFullDayNames(false);
+        } else {
+            setIsFullDayNames(true);
+        }
```

Comment:
setIsFullDayNames(Math.abs(document.querySelector("body").getBoundingClientRect().width > 768))

Reply:
I think it should be done by css - text-transform